### ISSUE TITLE
Problem with ellipsis in quotes

### ISSUE THREAD

**Issue by sehyoun** [sphinx-contrib/matlabdomain#71]
Sphinx crashes with the error message of 

```

Exception occurred:
  File "/some/path/to/python/lib/python2.7/site-packages/future/types/newlist.py", line 83, in __getitem__
    return super(newlist, self).__getitem__(y)
IndexError: list index out of range

```

when you have an ellipsis in a quoted text, e.g,

`disp('press any to continue...');`

An obvious work around is to use two periods.

I noticed the problem when I moved from v0.2.8 to v0.3.0 if that helps in narrowing things down.

**Comment by joeced** [comment]
Thanks for writing the issue. Do you know if this occurs with both Python 2.7 and 3? Further, do you have a minimal example that causes this error.

**Comment by sehyoun** [comment]
This was Python 2.7, but I can not speak for Python 3. I actually just keep python 2.7 for matlabdomain, so thank you for making it work with Python3 :+1:  I will try to create a minimal example.

**Comment by joeced** [comment]
Fixed in [version 0.3.3](https://github.com/sphinx-contrib/matlabdomain/releases/tag/0.3.3)
